src/header.jl

```diff
@@ -193,6 +193,21 @@ end
 # -----------------------------------------------------------------------------
 # Public API
 
+
+"""
+    read_key(hdu, key)
+
+Read the specified key and return a tuple of `(value, comment)`.
+
+The key, can be either the index of the header record (Integer)
+or the header keyword (ASCIIString).
```

Comment:
`ASCIIString` has been removed in julia 0.5, use `String` instead. There are also other uses of `ASCIIString`. See also #75.